### PR TITLE
Add Logout button to the menu bar

### LINKED ISSUE #68: [Add] Logout button to the menu bar
Add logout button to the nav bar on top for easy access to the user and design.

### PR DESCRIPTION
**Pull request by maraPislar**
I added to the frontend the **following changes**:

- added logout button to the menu bar
- added some text to the login page

**Preview**:

NOTE: normal button

NOTE: When hovering over the button

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1 +1,47 @@
 # GPostCard
+
+### Project Description
+
+GPostcard is a Web application that can generate postcards based on the text that is on them. After a postcard is generated, it can be sent to any Gmail user. By ticking the location, the user can get on the postcard a representative photo for their current location. Moreover, the user can record a message which will be automatically filled into the text area allowing accessability.
```
Comment (Krechals) [inline README.MD]: There is this PR https://github.com/googleinterns/step230-2020/pull/67 that deals with the README file. Let's focus there on this file.

**Review by Krechals — CHANGES_REQUESTED** [review]

File: README.MD (hunk 1 above)
Comment (maraPislar) [reply README.MD]: Oops, of course. Sorry!

**Review by Krechals — APPROVED** [review]

**Review by AndreMB0 — CHANGES_REQUESTED** [review]
Nice, but since the login is via SSO, it would be useful to have the userEmail written as well (above/below/next to the logout button in the menu bar)

**Comment by maraPislar** [comment]
I added the email next to the Logout link. Also added a left boarder for separating the links and for design.
**Preview**:

Screenshot 2020-09-18 at 8 14 20 PM

**Review by AndreMB0 — APPROVED** [review]

**Pull request merged**